src/pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1290,6 +1323,7 @@
       "BaselineVersion": "4.5.0",
       "InboxOn": {
         "netcoreapp3.0": "4.0.0.0",
+        "netcoreapp5.0": "5.0.0.0",
         "net45": "4.0.0.0",
         "uap10.0.16300": "4.0.0.0"
```

Comment:
Mop-up: we can delete all these entries.  @ViktorHofer

Reply:
https://github.com/dotnet/corefx/pull/42537